core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -279,13 +279,13 @@ class RequestSendThread(val controllerId: Int,
 
 }
 
-class ControllerBrokerRequestBatch(controller: KafkaController) extends  Logging {
+class ControllerBrokerRequestBatch(controller: KafkaController, eventManager: ControllerEventManager) extends  Logging {
```

Comment:
Good point. Previously I think my IDE tells me it can not be accessed as `controller.eventManager`. I have updated it to use `controller.eventManager`.